avm/res/container-service/managed-cluster/main.json

```diff
@@ -1270,6 +1270,20 @@
         "description": "Optional. Whether to enable Azure Defender."
       }
     },
+    "enableImageCleaner": {
+      "type": "bool",
+      "defaultValue": false,
+      "metadata": {
+        "description": "Optional. Whether to enable Image Cleaner for Kubernetes."
+      }
+    },
+    "imageCleanerIntervalHours": {
```

Comment:
Must be refreshed

Reply:
I will get this updated tomorrow when I get back on my laptop. Thanks for being patient.

Reply:
All good. Thanks for putting in the work 💪

Reply:
sset-avmmodules has been reran and changes pushed. this should be ready to go now.